src/main/java/io/cryostat/agent/remote/RecordingsContext.java

```diff
@@ -123,7 +123,11 @@ private static class RecordingInfo {
             this.duration = this.isContinuous ? 0 : rec.getDuration().toMillis();
             this.toDisk = rec.isToDisk();
             this.maxSize = rec.getMaxSize();
```

Comment:
Does this not happen with max size or any of the other options?

Reply:
Good call, it can happen to `startTime` if the recording has been created but not yet started. I think the others are all either non-nullable types - `maxSize` is a `long` - or have API docs saying they are not null (like the `name`).

Reply:
Can state be null?

Reply:
The `recording.getState()` has a javadoc saying it's `not null`, and it's an enum type so the `.name()` will also be not null.

Reply:
https://docs.oracle.com/en/java/javase/11/docs/api/jdk.jfr/jdk/jfr/Recording.html#getState()

Reply:
Sounds good